### PR TITLE
ref(api): Segment API endpoints

### PR DESCRIPTION
**Pull request by evanpurkhiser**
We ran out of arguments to pass to patterns. This should be more efficient anyway.

Also runs isort to sort imports consistently.

### CODE REVIEW

File: src/sentry/api/urls.py [hunk 1]
```diff
@@ -333,1067 +356,1085 @@
...
+    url(r'^organizations/', include([
+        url(r'^$', OrganizationIndexEndpoint.as_view(), name='sentry-api-0-organizations'),
+        url(
+            r'^(?P<organization_slug>[^\/]+)/$',
+            OrganizationDetailsEndpoint.as_view(),
+            name='sentry-api-0-organization-details'
+        ),
+
+        # Incidents
+        url(
```
Comment (wedamija) [inline src/sentry/api/urls.py]: Does it make sense to include incidents as a sub pattern? ( I assume you can have `^(?P<organization_slug>[^\/]+)/incidents/` as another level)

**Review by wedamija — COMMENTED** [review]
Looks good in general, probably need to test with getsentry as well just to be sure?

**Comment by EvanPurkhiser** [comment]
https://travis-ci.com/getsentry/getsentry/builds/118535665 getsentry tests running here.

File: src/sentry/api/urls.py (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/api/urls.py]: Yeah we could totally do more levels I think. I just didn't.

**Review by markstory — APPROVED** [review]

**Pull request merged**